### ISSUE TITLE
Editor Tools - more presets

### ISSUE THREAD

**Issue by pedromorgan** [ts2/ts2#58]
Just got an idea, whether its possible to include more "preset"..
and example would be a "junction".., and possibilities, in a drop down eg

Clicking a junction with icon
```
      ----------
     /
-----------------
```
Would show permutations
```
      ----------
     /
-----------------
```
```
------------------
     \
      ------------
```
```
-----------------
           /
-----------
```
```
-----------
           \
-----------------
```

The same could be done for crrossings, etc with presets for parallel crossovers, Y junctions etc..

**Comment by npiganeau** [comment]
Yes, we can imagine having presets being a combination of several trackitems already pre-configured.

To pull the idea further, it would be cool that the user can define their own presets by saving a selection of trackitems on the scenery.

**Comment by pedromorgan** [comment]
This is why we need the online server to share, and creaet signals et all and download and uplaod etc..

lets go.. mad idea but possible..

**Comment by pedromorgan** [comment]
see #47 for SVG icons idea
